pages/hobbies.css

```diff
@@ -0,0 +1,40 @@
+ul {
+    list-style-type: none; 
+    display: flex;
+    flex-wrap: wrap;
+    padding-left: 0px;
+}
+
+li {
+    width: 32%;
+    height: 100px;
+    line-height: 95px;
+    border: 3px solid #ed5b01;
+    text-align: center;
+}
+
+li:nth-child(1) {
```

Comment:
Is there a way to block nth child grouping in a cleaner way?

To save some lines on the code.